packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -660,6 +758,36 @@ class SyncWatcher extends BaseWatcher {
     await this.populateTransferRootTransferIds(transferRootId)
   }
 
+  async populateTransferSentToL2Timestamp (transferId: string) {
```

Comment:
I think this whole function can be removed? looks to be the same as `populateTransferSentTimestamp`